### PR TITLE
Issue #sl-survey : test : test case fix

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: projects/collection-editor-library/src/lib/components/editor/editor.component.ts [hunk 1]
```diff
@@ -832,10 +825,6 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
             field.options = this.setEvidence;
             field.range = null;
           }
-          else if (field.code === 'ecm') {
```
Comment (kiranharidas187) [inline projects/collection-editor-library/src/lib/components/editor/editor.component.ts:835]: is this not required?

File: projects/collection-editor-library/src/lib/components/editor/editor.component.ts [hunk 2]
```diff
@@ -915,10 +902,6 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
               field.options = this.setEvidence;
               field.range = null;
             }
-            else if (field.code === 'ecm') {
```
Comment (kiranharidas187) [inline projects/collection-editor-library/src/lib/components/editor/editor.component.ts:918]: is this not required?

Comment (pradeepkumarcm96) [reply projects/collection-editor-library/src/lib/components/editor/editor.component.ts:918]: ecm is only required for root level
for section level it is not required

File: projects/collection-editor-library/src/lib/components/editor/editor.component.ts (hunk 1 above)
Comment (pradeepkumarcm96) [reply projects/collection-editor-library/src/lib/components/editor/editor.component.ts:835]: ecm is only required for root level
for section level it is not required

**Pull request merged**